.github/workflows/rpc-tests.yml

```diff
@@ -33,20 +36,26 @@ jobs:
       - uses: actions/checkout@v4
       - uses: stellar/actions/rust-cache@main
       - run: rustup update
-      - run: sudo apt update && sudo apt install -y libudev-dev libdbus-1-dev gnome-keyring
+      - run:
+          sudo apt update && sudo apt install -y libudev-dev libdbus-1-dev
+          gnome-keyring
```

Comment:
Will this work with a line break?? We always add `\`s but it'd be great to know we don't need to lol